app/src/components/market/common/additional_market_data/index.tsx

```diff
@@ -155,84 +196,222 @@ export const AdditionalMarketData: React.FC<Props> = props => {
       getAPY()
     }
   }, []) // eslint-disable-line react-hooks/exhaustive-deps
-  return (
-    <AdditionalMarketDataWrapper>
-      <AdditionalMarketDataLeft>
-        <AdditionalMarketDataSectionWrapper
-          href={`/#/24h-volume/category/${encodeURI(category)}`}
-          noColorChange={true}
-          noMarginLeft={true}
-        >
-          <IconCategory size={'24'} />
-          <AdditionalMarketDataSectionTitle>{category}</AdditionalMarketDataSectionTitle>
-        </AdditionalMarketDataSectionWrapper>
-        <AdditionalMarketDataSectionWrapper
-          data-arrow-color="transparent"
-          data-for="marketData"
-          data-tip={`This market uses the ${oracle} oracle which crowd-sources the correct outcome.`}
-          href={realitioUrl}
-          rel="noopener noreferrer"
-          target="_blank"
-        >
-          <IconOracle size={'24'} />
-          <AdditionalMarketDataSectionTitle>{oracle}</AdditionalMarketDataSectionTitle>
-        </AdditionalMarketDataSectionWrapper>
-        <AdditionalMarketDataSectionWrapper
-          data-arrow-color="transparent"
-          data-for="marketData"
-          data-tip={`This market uses ${arbitrator.name} as the final arbitrator.`}
-          href={arbitrator.url}
-          rel="noopener noreferrer"
-          target="_blank"
-        >
-          <IconArbitrator size={'24'} />
-          <AdditionalMarketDataSectionTitle>{arbitrator.name}</AdditionalMarketDataSectionTitle>
-        </AdditionalMarketDataSectionWrapper>
-        {context.networkId !== networkIds.XDAI && (
-          <AdditionalMarketDataSectionWrapper
-            data-arrow-color="transparent"
-            data-for="marketData"
-            data-tip={
-              curatedByDxDaoOrKleros
-                ? 'This Market is verified by DXdao or Kleros and therefore valid.'
-                : 'This Market has not been verified and may be invalid.'
-            }
-            hasMarginRight={true}
-            isError={!curatedByDxDaoOrKleros}
-          >
-            {curatedByDxDaoOrKleros ? <IconVerified size={'24'} /> : <IconAlert size={'24'} />}
-            <AdditionalMarketDataSectionTitle isError={!curatedByDxDaoOrKleros}>
-              {curatedByDxDaoOrKleros ? 'Verified' : 'Not Verified'}
-            </AdditionalMarketDataSectionTitle>
-          </AdditionalMarketDataSectionWrapper>
-        )}
+
+  if (window.screen.width > 552) {
```

Comment:
Is there a way make these changes without duplicated code, using media queries maybe?

@KadenZipfel @pRoy24 @Mi-Lan thoughts?

Reply:
Yeah i agree !

Reply:
You can use media queries with styled components.
https://jsramblings.com/how-to-use-media-queries-with-styled-components/